app/utils/callback/lava.py

```diff
@@ -235,6 +236,36 @@ def _add_boot_log(meta, log, base_path):
         utils.lava_log_parser.run(log, meta, txt, html)
 
 
+def store_raw_json(job_data, base_path=utils.BASE_PATH):
+    """ Save the json callback object
+
+    Save LAVA v2 callback data as json file.
+
+    :param job_data: The JSON data from the callback.
+    :type job_data: dict
+    :param base_path: Path to the top-level directory where to store the files.
+    :type base_path: string
+    """
+
+    utils.LOG.info("Saving raw LAVA v2 callback data in {}".format(base_path))
+    job_definition = yaml.load(job_data["definition"], Loader=yaml.CLoader)
+    file_name = job_definition["job_name"] + ".json"
+    file_path = os.path.join(base_path, "json-raw", file_name)
```

Comment:
My only concern here is with the filesystem path. Why not storing them at the same path where the boot log (HTML and TXT) files are stored?

I understand is more practical to get all the JSON files from one directory, but we also have to deal with that directory on server side and clean it up from time to time (if the file is stored along with the boot logs, it gets clean automatically after 40 days).

Reply:
BTW, a long term plan (briefly discussed) is to open the data base data and release it publicly (like every 6 months). It's still up for discussion, we don't know the license etc... but it could also solve part of the problem of grabbing some data to work with.

Reply:
OK I will move it to be stored along the boot logs. 

Getting the raw JSON would already be a nice first step in sharing the data :)
Restoring the database would also be useful, we could use it with docker then.